dot/services_integration_test.go

```diff
@@ -180,42 +187,43 @@ func TestCreateRPCService(t *testing.T) {
 		network:     networkSrvc,
 		system:      sysSrvc,
 	}
-	rpcSrvc, err := createRPCService(rpcSettings)
+	rpcSrvc, err := nodeInstance.createRPCService(rpcSettings)
 	require.NoError(t, err)
 	require.NotNil(t, rpcSrvc)
 }
 
-func TestCreateBABEService(t *testing.T) {
+func TestCreateBABEService_Integration(t *testing.T) {
 	cfg := NewTestConfig(t)
 
-	genFile := NewTestGenesisRawFile(t, cfg)
+	genFile := newTestGenesisRawFile(t, cfg)
 
 	cfg.Core.Roles = types.FullNodeRole
 	cfg.Init.Genesis = genFile
 
 	err := InitNode(cfg)
 	require.NoError(t, err)
 
-	stateSrvc, err := createStateService(cfg)
+	nodeInstance := nodeBuilder{}
```

Comment:
rename to `builder` or `nodeBuilder`, it's not really a `nodeInstance`... maybe a `nodeBuilderInstance` if you really want to 😄

Reply:
Updated.